src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -1801,6 +1911,29 @@ bool WindowImplX11::processEvent(XEvent& windowEvent)
     return true;
 }
 
+
+////////////////////////////////////////////////////////////
+bool WindowImplX11::checkXRandR(int &xRandRMajor, int &xRandRMinor)
+{
+    // Check if the XRandR extension is present
+    int version;
+    if (!XQueryExtension(m_display, "RANDR", &version, &version, &version))
+    {
+        // XRandR extension is not supported
+        err() << "XRandR extension is not supported" << std::endl;
+        return false;
+    }
+
+    // Check XRandR version, 1.2 required
+    if(!XRRQueryVersion(m_display, &xRandRMajor, &xRandRMinor) || xRandRMajor < 1 || xRandRMinor < 2 )
+    {
+        err() << "XRandR is to old" << std::endl;
```

Comment:
s/to/too/

Check will fail once v.2 is release (although it's not for tomorrow).